docs/sources/userguide/usingdocker.md

```diff
@@ -68,13 +68,8 @@ You will see a list of all currently available commands.
 
 You can also zoom in and review the usage for specific Docker commands.
 
-Try typing Docker followed with a `[command]` to see the usage for that
-command:
-
-    $ docker attach
-    Help output . . .
-
-Or you can also pass the `--help` flag to the `docker` binary.
+Try typing a Docker command followed with flag `--help` to see the usage for
```

Comment:
This sentence is missing an article.  Also, well, the whole section reads a bit oddly. I wonder if you'd be willing to expand the scope a bit to clean up a few things:

-----> https://gist.github.com/moxiegirl/fb4d11255b598884f892

### Get Docker command help

You can display the help for specific Docker commands. The help details the
options and their usage. To see a list of all the possible commands, use the
following:

		$ docker --help

To see usage for a specific command, specify the command with the `--help` flag:

    $ docker attach --help

		Usage: docker attach [OPTIONS] CONTAINER

		Attach to a running container

			--help=false        Print usage
			--no-stdin=false    Do not attach STDIN
			--sig-proxy=true    Proxy all received signals to the process

For further details and examples of each command, see the  [command
reference](/reference/commandline/cli/) in this guide.